package.json

```diff
@@ -52,7 +52,7 @@
     "opn": "5.1.0",
     "os-locale": "^2.0.0",
     "prettyjson": "^1.2.1",
-    "ramda": "^0.24.1",
+    "ramda": "0.25.0",
```

Comment:
Did you check the `CHANGELOG` and ensure nothing is breaking?

Reply:
Nope, go XHG or go home.